### ISSUE TITLE
in POST v2/subscriptions if httpCustom url is empty the subcription is created

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2279]
in POST v2/subscriptions if notification **httpCustom** url is empty the subcription is created
#### subscription request

```
POST http://qa-orion-fe-02:1026/v2/subscriptions
headers:
    Content-Type: application/json
    Fiware-Service: test_notif_http_custom_url_error
    Fiware-ServicePath: /test
payload: {"notification": {"httpCustom": {"url": ""}, "attrs": ["temperature"]}, "expires": "2016-04-05T14:00:00.00Z", "subject": {"entities": [{"idPattern": ".*"}], "condition": {"attrs": ["temperature"]}}}
```
#### subcription response

```
http code: 201 - Created
headers:
   date: Wed, 15 Jun 2016 11:14:09 GMT
   fiware-correlator: 47e3b3d0-32ea-11e6-be09-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/57613940451e58b1a26c24b2
```
#### mongo doc

```
{ "_id" : ObjectId("57613940451e58b1a26c24b2"), "expiration" : NumberLong(1459864800), "reference" : "", "custom" : true, "throttling" : NumberLong(0), "servicePath" : "/test", "status" : "active", "entities" : [ { "id" : ".*", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
```
#### expected response

```
http code: 400 - Bad Request
payload: {"error":"BadRequest","description":"Invalid URL parsing notification url"}

```

**Comment by iariasleon** [comment]
Re-tested in the CB version. It issue still fails and returns `201-Created`
```
  "version" : "1.2.0-next",
  "git_hash" : "d81dcf28e71d8d63dc0929ef7a6b73d7db47421b"
```

**Comment by fgalan** [comment]
Fixed in PR #2815. Please @iariasleon check (once the PR gets merged).

**Comment by iariasleon** [comment]
CB version used:
```
  "version" : "1.6.0-next",
  "git_hash" : "6232159b397f40aff97abd497b1e142a510e3e6d",
```

LGTM
```
http code: 400 - Bad Request
headers:
   Connection: Keep-Alive
   Content-Length: 59
   Content-Type: application/json
   Fiware-Correlator: 7d5f872c-dbf1-11e6-8d21-005056a20feb
   Date: Mon, 16 Jan 2017 13:41:31 GMT
payload: {"error":"BadRequest","description":"invalid custom /url/"}
```
